storybook/src/components/Dialog/Dialog.docs.mdx

```diff
@@ -21,3 +21,10 @@ Content taller than the dialog itself will scroll.
 Click or tap the button to open a dialog.
 
 <Canvas of={DialogStories.TriggerButton} />
+
+## Vertically Stacked Buttons
+
+If the buttons don’t fit next to each other, they will stack vertically and stretch to the full width.
+View this example in a narrow window, enlarge your browser’s text size, or [use the ‘Phone’ size of the preview](?path=/story/components-containers-dialog--vertical-buttons&globals=viewport:phone).
```

Comment:
Can't we set the inline story to be less wide? So the user doesn't have to resize their window?

Reply:
Can’t find a way. There is only a `height` prop on `docs`, not `width`.

Setting a width on the dialog through an inline style strangely makes the button labels wrap, breaking the example.

Reply:
Nah, screenshots are static and age quickly.

The current setup immediately shows the vertical stacking, let’s leave it as is.

Reply:
We could make the button labels longer?

Reply:
I agree, but you'll need some long button text.